recipes/cppcommon/all/conandata.yml

```diff
@@ -0,0 +1,17 @@
+sources:
+  "1.0.0.0":
+    url: "https://github.com/chronoxor/CppCommon/archive/1.0.0.0.tar.gz"
+    sha256: "d6124dd4cd430e5f10c2942ff02a95636369875b652e66a499b01f1fb3ecfd6d"
+  "19265a8d721e9c8d5f706c11afbd8885fa9ee9e2":
```

Comment:
The release 1.0.0 was 18 days ago, I don't think adding a custom version now is a good idea. Ask for a new release if you really need a new fix/feature.

Reply:
This project actually doesn't have an usual release cycle. The author keeps commiting in the master branch and the pipelines check the whole thing. This 1.0.0.0 version was added after I opened an issue (https://github.com/chronoxor/CppCommon/issues/4).

I'm thinking that maybe we should stick to a different versioning scheme here on conan center that is more suited to the way the project handles its own versioning.

In [another PR of mine](#3380), @SpaceIm suggested something I didn't know: to use a version number like this `cci.20171104` to represent _the version_ that was up on the date 2017 11 04.

For instance, to get the 1.0.0.0 version of this repo working properly with all the pipelines covered here on conan center, it needs some patching. I think it would be better to skip this version altogether and go along with the way the author updates upstream, selecting some specific _checkpoints_ from which a version is generated.

In this case, the version `cli.20201029` (which is 7 commits ahead of 1.0.0.0) would need only a patch on the `CMakeLists.txt` to integrate with Conan. Although the 1.0.0.0 version needs some non trivial bug fixing.

As the author itself doesn't seem to care to some traditional versioning scheme, isn't it better to use the `cli.20201029` approach instead?

Reply:
Please, read: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

If cppcommon offers an official release, it should be added in conandata.yml, only if it's broken you could avoid, but otherwise it should be added.

And, either the upstream doesn't offer regular releases, because prefers bleeding-edge versioning, use the recommended version which is documented on FAQs.

Reply:
Fixed in 2e6b9154, I think...